test/parallel/test-fs-fsync.js

```diff
@@ -1,32 +1,22 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
 
-var path = require('path');
-var fs = require('fs');
-var successes = 0;
+const path = require('path');
+const fs = require('fs');
 
 var file = path.join(common.fixturesDir, 'a.js');
 
-fs.open(file, 'a', 0o777, function(err, fd) {
+fs.open(file, 'a', 0o777, common.mustCall(function(err, fd) {
   if (err) throw err;
```

Comment:
Actually, you could change the other occurrences of `if (err) throw err;` as well.